jekyll/_cci2/releases/set-up-a-release-integration.adoc

```diff
@@ -0,0 +1,110 @@
+---
+contentTags:
+  platform:
+  - Cloud
+---
+= Set up a release integration - Open preview
+:page-layout: classic-docs
+:page-liquid:
+:page-description: Tutorial outlining how to set up a new CircleCI release environment and install the CircleCI release agent.
+:icons: font
+:experimental:
+
+NOTE: CircleCI releases is currently in **open preview**
+
+Visualise and control deployments to your Kubernetes cluster with CircleCI release environments. The steps outlined on this page guide you to set up a release environment and install the CircleCI release agent into your Kubernetes cluster.
+
+[#introduction]
+== Introduction
+
+In this tutorial you will set up an integration between CircleCI and your Kubernetes cluster. Once configured, release status can be reported to the CircleCI web app, and commands from CircleCI can be #mapped to your release ???# to enable you to:
+
+* Restore a version
+* Scale a version
+* Restart a version
+* Retry a release (when using Argo Rollouts)
+* Promote a release (when using Argo Rollouts)
+* Cancel a release (when using Argo Rollouts)
+
+[#prerequisites]
+== Prerequisites
+
+In order to set up a release integration with CircleCI you will need to ensure the following prerequisites are fulfilled:
+
+* A CircleCI account connected to your code. You can link:https://circleci.com/signup/[sign up for free].
+* A project building on CircleCI. See the xref:../create-project#[Create a project] page for steps.
+* You must be an **org admin**. For full details of roles and permissions for GitLab and GitHub App integrations, see the xref:../roles-and-permissions-overview.adoc#[Roles and permissions overview]. If your integration is through Bitbucket or the GitHub OAuth app, you will need to be an org admin in Bitbucket/GitHub respectively. To find out which GitHub integration you have, see the xref:../github-apps-integration.adoc#[GitHub App integration] page.
+* A Kubernetes .... #Not sure of the specifics here - they need a Kubernetes cluster to which they are deploying something? Or should we say a Kubernetes Deployment?#
+
+CircleCI supports Kubernetes, Helm, and Argo Rollouts. The supported versions are listed below:
+
+[.table.table-striped]
+[cols=2*, options="header", stripes=even]
+|===
+|System
+|Supported versions
+
+|Kubernetes
+a| 1.28.0 +
+1.27.0
+
+|Helm (optional)
```

Comment:
Minor clarification: We currently only support installation of the agent via Helm, but we don't require customers to use Helm to deploy their components.